apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -816,7 +816,7 @@ const CTA = ({ data }: { data: GetByViewerResponse }) => {
 const Actions = () => {
   return (
     <div className="hidden items-center md:flex">
-      <TeamsFilter popoverTriggerClassNames="mb-0" showVerticalDivider={true} />
+      <TeamsFilter useProfileFilter popoverTriggerClassNames="mb-0" showVerticalDivider={true} />
```

Comment:
It is a way to opt-in to profile based DB querying. As event-type supports profileId now.